unity/Assets/Scripts/Quest/CameraController.cs

```diff
@@ -177,6 +196,13 @@ void Update()
             {
                 // How many units to move this frame
                 float moveDist = Time.deltaTime * autoPanSpeed;
+
+                if (game.editMode)
+                {
+                    // in edit mode, there is pan but not zoom
+                    camTarget.z = gameObject.transform.position.z;
```

Comment:
I think this would make more sense where the target is set further down:

 cc.camTarget = new Vector3(pos.x, pos.y, -8);

Reply:
But this will cause zoom again if I have the zoom totally out, the usual zoom when you want to see whole board.
-8 is the closest hight no?
Don't agree with not zooming in edit mode?

Reply:
The end result is fine, I just mean that it should be changed where the position is set rather than later.

So in editMode:
cc.camTarget = new Vector3(pos.x, pos.y, cc.camTarget.z);

rather than:
cc.camTarget = new Vector3(pos.x, pos.y, -8);

Reply:
am, ok. changing